src/extension/wps/wps-core/src/main/java/org/geoserver/wps/GetExecutionsTransformer.java

```diff
@@ -0,0 +1,809 @@
+                                                start("wfs:GetFeature", getFeatureAttributes);
+                                                EList queries = features.getQuery();
+                                                if (queries != null && !queries.isEmpty()) {
+                                                    for (Object query : queries) {
+                                                        QueryType qt = (QueryType) query;
+                                                        final AttributesImpl queryTypeAttributes =
+                                                                new AttributesImpl(inputAttributes);
+                                                        if (qt.getTypeName() != null
+                                                                && !qt.getTypeName().isEmpty()) {
+                                                            queryTypeAttributes.addAttribute(
+                                                                    "",
+                                                                    "typeName",
+                                                                    "typeName",
+                                                                    "",
+                                                                    qt.getTypeName()
+                                                                            .get(0)
+                                                                            .toString());
+                                                        }
+                                                        start("wfs:Query", queryTypeAttributes);
+                                                        Filter filter = qt.getFilter();
+                                                        if (filter != null) {
+                                                            FilterPPIO fppio =
+                                                                    new FilterPPIO.Filter11();
+                                                            fppio.encode(filter, contentHandler);
+                                                        }
+                                                        end("wfs:Query");
+                                                    }
+                                                }
+                                                end("wfs:GetFeature");
+                                            } else {
+                                                element(
+                                                        "wps:ComplexData",
+                                                        "<![CDATA[" + out.toString() + "]]",
+                                                        inputAttributes);
+                                            }
+                                        } catch (Exception e) {
+                                            // LOGGER.log(Level.WARNING, "", e);
```

Comment:
Please don't eat exceptions. The formatting should also suggest you to roll the encoding of the specific inputs into a separate method, and do a catch around it.